pydata_sphinx_theme/webpack-macros.html

```diff
@@ -0,0 +1,20 @@
+<!-- these macros are generated by "yarn build". do not edit by hand. -->
+{% macro head_pre() %}
+  <link rel="stylesheet"
+    href="{{ pathto('_static/vendor/fontawesome/5.13.0/css/all.min.css', 1) }}">
```

Comment:
these version numbers are extracted in `webpack.common.js` from their respective `package.json`, so won't need to be hand-maintained.

Reply:
Webpack reads this file and updates the version number?

Reply:
Nevermind, I see the update in `webpack.common.js`.